hera_mc/cm_transfer.py

```diff
@@ -136,8 +140,8 @@ def __initialization(args):
             print('Initialization for %s not found' % (table))
             use_table.remove(table)
         else:
-            if args.maindb:
-                if dbkey != args.maindb:
+            if maindb:
+                if dbkey != maindb:
```

Comment:
Oh, seeing what this `maindb` variable is ... I do not think it has a very good name. But that's an issue for another PR.